staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -542,19 +542,19 @@ type CreateOptions struct {
 
 	// fieldValidation instructs the server on how to handle
 	// objects in the request (POST/PUT/PATCH) containing unknown
-	// or duplicate fields, provided that the `ServerSideFieldValidation`
-	// feature gate is also enabled. Valid values are:
+	// or duplicate fields. Valid values are:
 	// - Ignore: This will ignore any unknown fields that are silently
 	// dropped from the object, and will ignore all but the last duplicate
 	// field that the decoder encounters. This is the default behavior
-	// prior to v1.23 and is the default behavior when the
-	// `ServerSideFieldValidation` feature gate is disabled.
+	// prior to v1.23 or prior to v1.27 when the `ServerSideFieldValidation`
+	// feature gate is disabled.
 	// - Warn: This will send a warning via the standard warning response
 	// header for each unknown field that is dropped from the object, and
 	// for each duplicate field that is encountered. The request will
 	// still succeed if there are no other errors, and will only persist
 	// the last of any duplicate fields. This is the default when the
-	// `ServerSideFieldValidation` feature gate is enabled.
+	// `ServerSideFieldValidation` feature gate is enabled (v1.23+) or
+	// locked to enabled in GA (v1.27+)
```

Comment:
I would also simplify this:

```suggestion
	// the last of any duplicate fields. This is the default in v1.23+.
```

Reply:
same comment applies below

Reply:
done